### PR TITLE
Algorithm tracking

### PR DESCRIPTION
**Pull request by chribaue**
ENH: functionality to specify DICOM algorithm name and type and export to DICOM SEG

### CODE REVIEW

**Comment by che85** [comment]
@chribaue Just out of curiosity, what if the user selects a tool, uses it to edit a segment and then decides to undo the changes and select another tool?

File: DICOMPlugins/DICOMSegmentationPlugin.py [hunk 1]
```diff
@@ -205,6 +205,10 @@ def load(self,loadable):
           labelNode.SetAttribute("ColorR", str(rgb[0]))
           labelNode.SetAttribute("ColorG", str(rgb[1]))
           labelNode.SetAttribute("ColorB", str(rgb[2]))
+          if "SegmentAlgorithmType" in segment:
+            labelNode.SetAttribute("SegmentAlgorithmType", segment["SegmentAlgorithmType"])
+          if "SegmentAlgorithmName" in segment:
+            labelNode.SetAttribute("SegmentAlgorithmName", segment["SegmentAlgorithmName"])
```
Comment (fedorov) [inline DICOMPlugins/DICOMSegmentationPlugin.py]: Is there a good reason to not use `DICOM.` prefix here, but then use it later while assigning attributes to the `segmentLabelNode`? Shouldn't it be consistent?

**Comment by fedorov** [comment]
@chribaue this looks good to me, just one minor comment.

File: DICOMPlugins/DICOMSegmentationPlugin.py (hunk 1 above)
Comment (chribaue) [reply DICOMPlugins/DICOMSegmentationPlugin.py]: `labelNode` is just temporary data, but I'll add the `DICOM.` prefix for readability

Comment (fedorov) [reply DICOMPlugins/DICOMSegmentationPlugin.py]: Great, thanks - I have no other concerns.

**Pull request merged**